packages/babel-preset-typescript/test/normalize-options.spec.js

```diff
@@ -83,6 +87,7 @@ describe("normalize options", () => {
           "jsxPragma": undefined,
           "jsxPragmaFrag": "React.Fragment",
           "onlyRemoveTypeImports": undefined,
+          "optimizeConstEnums": undefined,
```

Comment:
nit: we could enforce type checking of `optimizeConstEnums` in Babel 7. It is a new option.